force-app/main/default/lwc/bdiFieldMappings/bdiFieldMappings.js

```diff
@@ -72,7 +71,6 @@ export default class bdiFieldMappings extends LightningElement {
         bdiFMUINoFieldMappings,
         bdiFMUISourceObject,
         bdiFMUITarget,
-        stgHelpAdvancedMapping3,
```

Comment:
@andrewyu-salesforce / @daniel-fuller I removed all values/references to `stgHelpAdvancedMapping3` based on the POU audit sheet, should I "reindex" the label value for `stgHelpAdvancedMapping4` and make it `stgHelpAdvancedMapping3` to maintain consecutive consistency, or leave it as is?

Reply:
@balsamhindi If that label doesn't need to be replaced with another one, I would just leave it as is